### ISSUE TITLE
Turn off non-compiled code highlighting of slime

### ISSUE THREAD

**Issue by gaya-** [code-iai/ros_emacs_utils#10]
On dark background it's really annoying. This should be an optional feature IMO. As slime developers don't think the same we should turn it off in slime-ros instead.

`(slime-highlight-edits-mode 0)`

**Comment by fairlight1337** [comment]
Nah, its okay for me to change the default. I can just change it for me locally if it annoys you. But making it depend on the background is a good idea as well.

**Comment by gaya-** [comment]
There is a commented out line for people who would like to turn off the feature. So closing the issue.
